xbmc/utils/Observer.cpp

```diff
@@ -24,72 +24,20 @@
 
 #include <algorithm>
 
-Observer::~Observer(void)
-{
-  StopObserving();
```

Comment:
@paxxi: This is imo a bad idea, because now no longer is guaranteed that on destruction observers are removed from their observable. Thus Observable::NotifyObservers may call observers which are already destructed. This may be the cause of the crash reported here http://forum.kodi.tv/showthread.php?tid=269814&pid=2388702#pid2388702.

... meanwhile I got a similar crash which seems to confirm my assumption: http://pastebin.com/sQsMHCDi